core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksBlockStore.java

```diff
@@ -49,11 +50,12 @@
   private static final String BLOCK_META_COLUMN = "block-meta";
   private static final String BLOCK_LOCATIONS_COLUMN = "block-locations";
 
-  private final String mBaseDir;
+  // This is a field instead of a constant because it depends on the call to RocksDB.loadLibrary().
+  private final WriteOptions mDisableWAL;
+
+  private final RocksStore mRocksStore;
```

Comment:
Now that we have `mRocksStore`, do we need `mDb`?

It feels a bit weird to have the two member variables possibly out of sync.

Reply:
Updated to set columns via `AtomicReference` and always fetch the latest `RocksDB` via `RocksStore.getDb()`